tools/sil-func-extractor/SILFunctionExtractor.cpp

```diff
@@ -250,6 +255,15 @@ int main(int argc, char **argv) {
   Invocation.getLangOptions().EnableAccessControl = false;
   Invocation.getLangOptions().EnableObjCAttrRequiresFoundation = false;
 
+  if (EnableObjCInterop == llvm::cl::BOU_UNSET) {
+    Invocation.getLangOptions().EnableObjCInterop =
+        Invocation.getLangOptions().Target.isOSDarwin();
+  } else if (EnableObjCInterop == llvm::cl::BOU_TRUE) {
+    Invocation.getLangOptions().EnableObjCInterop = true;
+  } else {
+    Invocation.getLangOptions().EnableObjCInterop = false;
+  }
```

Comment:
I think that this would be simpler as:

```suggestion
  } else {
    Invocation.getLangOptions().EnableObjCInterop =
        EnableObjCInterop == llvm::cl::BOU_TRUE;
  }
```